bigquery_etl/copy_deduplicate.py

```diff
@@ -254,8 +254,26 @@ def _copy_join_parts(client, stable_table, query_jobs):
             job_config = bigquery.CopyJobConfig(
                 write_disposition=bigquery.WriteDisposition.WRITE_TRUNCATE
             )
-            copy_job = client.copy_table(sources, stable_table, job_config=job_config)
+
+            # workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1722546
+            # split copy_table() jobs
+            tmp_table_1 = temp_dataset.temp_table()
+            copy_job_1 = client.copy_table(
+                sources[: len(sources) // 2], tmp_table_1, job_config=job_config
+            )
+            copy_job_1.result()
+
+            tmp_table_2 = temp_dataset.temp_table()
+            copy_job_2 = client.copy_table(
+                sources[len(sources) // 2 :], tmp_table_2, job_config=job_config
+            )
+            copy_job_2.result()
+
+            copy_job = client.copy_table(
+                [tmp_table_1, tmp_table_2], stable_table, job_config=job_config
+            )
             copy_job.result()
+
             logging.info(f"Copied {len(query_jobs)} results to populate {stable_table}")
             for job in query_jobs:
                 client.delete_table(job.destination)
```

Comment:
need to clean up the intermediate tables:
```suggestion
            for job in query_jobs + [copy_job_1, copy_job_2]:
                client.delete_table(job.destination)
```